tests/benches/deadlock_detector/mod.rs

```diff
@@ -80,28 +82,29 @@ fn bench_dense_detect_without_cleanup(c: &mut Criterion) {
         10_000_000,
         100_000_000,
     ];
-    let mut cfgs = vec![];
     for range in ranges {
-        cfgs.push(Config {
+        let config = Config {
             n: 10,
             range,
             ttl: Duration::from_secs(100000000),
-        });
+        };
+        group.bench_with_input(format!("{:?}", &config), &config, bench_detect);
```

Comment:
This panics due to "Benchmark IDs must be unique within a group.". It's because the there are two `10_000` in the `ranges` array. :(

Reply:
Fixed, by removing one of them :cry: